src/lib/containers/ExperimentalMode.tsx

```diff
@@ -24,17 +26,40 @@ const ExperimentalMode: React.FC = () => {
   const createExperimentalModeCookie = () => {
     cookies.set(EXPERIMENTAL_MODE_COOKIE, { path: '/' })
     setIsExperimentalModeOn(true)
+    window.location.reload();
```

Comment:
I reload the browser after a user turns the experimental mode on/off. I think sometimes people don't know they have to do that in order to see the changes.